content/about/funding.md

```diff
@@ -0,0 +1,94 @@
+---
+$title@: Funding
+$order: 7
+---
+
+This page lists currently active funding for our projects
+
+#### Project Title: Synaptomes of Mouse and Man
```

Comment:
remove "Project Title" from each of the headings.

Reply:
Will do

Reply:
done